airflow-core/src/airflow/api_fastapi/core_api/datamodels/assets.py

```diff
@@ -42,6 +42,13 @@ class TaskOutletAssetReference(StrictBaseModel):
     updated_at: datetime
 
 
+class LastAssetEventResponse(BaseModel):
+    """Last asset event response serializer."""
+
+    id: int | None = None
```

Comment:
```suggestion
    id: NonNegativeInt | None = None
```

nit. I think we should use NonNegativeInt. Since it definitely shouldn't be a negative integer.